### ISSUE TITLE
Multiline Comments Before Method Decl Bork The Parser

### ISSUE THREAD

**Issue by mccartnm** [mccartnm/CppToolkitSublimeText#2]
```cpp
class foo {

public:
    /*
        bar
    */
    void baz(int schmoo);

};
```
Right clicking on that will bust up the parser and give you "generate getter/setter" rather than the declaration options we're looking for.

Need to have the line-finder build in a block-by-block fashion to assert that we're not in a comment for appending.

**Comment by mccartnm** [comment]
I'm throwing in a couple more glitches onto this bug.

```cpp
class foo {

public:
    void bezerk(); // a eol comment

    void baz(int schmoo);

};
```
Simple but explodes the parser too. Probably a good reason to rework the parsing of the lines all together.

**Comment by mccartnm** [comment]
This is fixed and will be out with 0.3.0
